packages/auth/src/tests/grant.ts

```diff
@@ -0,0 +1,57 @@
+import { faker } from '@faker-js/faker'
+import { v4 as uuid } from 'uuid'
+import { FinishMethod, Grant, GrantState, StartMethod } from '../grant/model'
+import { generateNonce, generateToken } from '../shared/utils'
+import { AccessAction, AccessType } from 'open-payments'
+import { GrantRequest } from '../grant/service'
+
+export async function createGrant(): Promise<Grant> {
```

Comment:
This could take `deps: IocContract<AppServices>` like `backend` [test helpers](https://github.com/interledger/rafiki/tree/main/packages/backend/src/tests)
https://github.com/interledger/rafiki/blob/e518e2c9960da4cbe69328b9281c152f2d758101/packages/backend/src/tests/peer.ts#L11-L16
which would allow calling
```
  const grantService = await deps.use('grantService')
  return await grantService.create({ /** minimal GrantRequest **/ })
```

In face, I think this would let you remove `getGrantRequest` below and replace existing `grantService.create(getGrantRequest())` with `createGrant`